todo_app/tests/unit_test.py

```diff
@@ -0,0 +1,25 @@
+
+import os,sys,inspect, pytest 
+from datetime import datetime
+sys.path.append('/srv/www/todo_app')
+
+class TestClass():
+    def setup_class(self):
+        pass
+
+    def teardown_class(self):
+        pass
+
+
+    def setup_method(self):
+        now = datetime.now()
+        self.current_time = now.strftime("%H:%M:%S")
+
+    def teardown_method(self):
+        pass # we will add some fancy webhooks and emails here passed on the status of the tests later
+
+    # Test function to add a card
+    def test_unittestWebServer(self,app, client):
+        result = client.get('/')
+        assert result.status_code == 200
```

Comment:
Not directly relevant here, but this is doing a lot for a unit test! It's preparing your app, triggering its index function and checking the response; this will trigger live hits to a database, so it's at least in the integration test region and I'd argue is really a simple end-to-end test!